### DISCUSSION TITLE
I don't want to put my name in my introduction.

### DISCUSSION THREAD

**Discussion by yama-can** [community/community#57486] [GitHub Education] [answered]
When I registered for GitHub Education, I got an error saying "Write a self-introduction in your profile".Is it possible to resolve the error without writing the name?

**Comment by samus-aran** [comment, marked as answer]
HI @yama-can 

Thank you for feedback and raising your concerns! We will pass this along to our internal team.

We are extremely proud that the GitHub Global Campus has provided millions of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 
However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. While submitting requested information is voluntary, we do thank students for providing this if possible as it speeds up their verification.

While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. One aspect of this protection is momentarily compare the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information as it's only required upon each new submission. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).
